server/src/models/calendar.js

```diff
@@ -1 +1,12 @@
-console.log('calendar.js');
\ No newline at end of file
+const mongoose = require('mongoose');
+
+const calendarSchema = new mongoose.Schema({
+  title: String,
```

Comment:
you should create a custom id field if needed to match the one from the frontend which is separate to _id